src/components/common/pop-over/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { Whisper } from "rsuite";
+import "rsuite/dist/rsuite.css";
+import { User } from "../../../interfaces/components/tweet-card/index";
+import { addUserToAnalysis } from "../../../store/modules/tweet/analysis/action";
+import { hashTagHighlighter } from "../../../utils/helper";
+import UserAvatar from "../avatar";
+import Title from "../title";
+import {
+  AddToAnalysisButton,
+  AvatarContainer,
+  Connections,
+  PopOverBody,
+  PopOverHeader,
+  TextContainer,
+  UserDetails,
+  UserDiscription,
+} from "./style";
+
+const avatarStyle = { height: "70px", width: "70px" };
```

Comment:
i have seen this avatar style in multiple places, why don't we make a const with default avatar style and use that constant?